docs/dimensions/values/temporal entity.md

```diff
@@ -0,0 +1,7 @@
+A special type of [[dimension value]], which has a character-based [[schema > position]] and uses shared values (e.g. https://ld.admin.ch/time/month/2000-02).
+
+Can appear in [[TemporalEntityDimension]].
+
+### assumptions
+
+- [[schema > position]] contains the full, formatted date – according to its [[time unit]]
```

Comment:
Nit: In general, I think prefer a bit larger documents with a table of contents.